e-picsa/picsa-apps
# Feat Enable Password Reset

## Description

This PR adds a feature to reset the password using superbase

## Discussion

Closes: #309

## Preview

_Link to app preview if relevant_

## How to test it ( include screenshots)
1. Run this branch locally
2. Make sure the server is running
3. Go to the dashboard
4. Signout
<img width="238" alt="Screenshot 2024-08-06 at 21 24 40" src="https://github.com/user-attachments/assets/c475be25-9f99-4558-82a8-12cf7fa46f40">

6. Create a new user if you dont have one
<img width="439" alt="Screenshot 2024-08-06 at 21 24 48" src="https://github.com/user-attachments/assets/96fd56df-34de-4b7b-9ad1-17c6448dcbbb">

8. Then for the user you have created initiate a reset password with there email
<img width="494" alt="Screenshot 2024-08-06 at 21 24 54" src="https://github.com/user-attachments/assets/d81730e2-b063-4d4e-8ddc-76a23d98e6bd">

10. An email will not be sent locally but rather you will find it under monitoring on the link:
http://localhost:54324/ among the running docker servicer, it will be under monitoring.
<img width="1449" alt="Screenshot 2024-08-06 at 21 31 25" src="https://github.com/user-attachments/assets/43561d94-9d61-4dd7-bb7d-cad7757e1263">
<img width="1452" alt="Screenshot 2024-08-06 at 19 36 06" src="https://github.com/user-attachments/assets/a9f2cfd6-2ac2-49a6-9f9f-d480bec051c2">

11. Click on the link to reset password and you will be redirected to a screen where you can enter the new password
<img width="1468" alt="Screenshot 2024-08-06 at 21 20 52" src="https://github.com/user-attachments/assets/712861aa-338c-4d45-9fe4-d638554bccd8">

12. On submission, you will be done reseting your password.
